README.md

```diff
@@ -1,8 +1,103 @@
 ![Momentum Mod](https://momentum-mod.org/assets/images/logo.svg)
 
-> *Momentum Mod is a standalone game built on the Source Engine, aiming to centralize movement gametypes found in CS:S, CS:GO, and TF2.*
+> *Momentum Mod is a standalone game built on the Source Engine, aiming to centralize movement gametypes found in CS:S,
+CS:GO, and TF2.*

```

Comment:
120 char line length?

Reply:
Yes, that's what I currently have configured, though I just noticed I wasn't setting `max_line_length` in our editorconfig (Roslyn doesn't use it so doesn't matter to us very much).

Are you against this value in particular? This works fine on Github, not sure what your point is!

Reply:
80 is the common one and the editor config in this commit was 100 and I just wanted to mention it because its a third number. I like 80 but whatever, no point arguing about this.

Reply:
I'm not trying to argue about this. Commenting "120 char line length?" just doesn't clearly convey what your point is.

Decided on formatting rules we like is worth trying to get out the way with now since this PR reformats everything, I'd you prefer we go down to 80 that's completely fine. Roslyn unfortunately [doesn't enforce a maximum line length](https://github.com/dotnet/roslyn/issues/47158) but we could just use 80 everywhere, could also including a Resharper formatting config for anyone using it.

I just set my max line length in Rider/Resharper to 80 and auto-formatted, was surprised by how good it looked, also since we use top-level namespaces now, we shave 4 spaces off most lines. That being said, I worry that without a strict autoformatter like Prettier, wrapping at 80 is likely to provoke more formatting disagreements in certain cases.

I'm not overly attached to any value, so long as we pick one and use it consistently. I'm most inclined to stick with 120 since it probably saves some formatting nitpicking, and saves me having to go through every file checking formatting again and fixing comment wrapping, but I've no problem doing that if you're prefer 80/100. Maybe best to discuss with others in Discord as well?
?

Reply:
Also, if we were to use 80 we'd probably only want to enforce it in C# files, and not XAML. Some files like EntityEditorView.axaml become pretty unreadable, and I don't like breaking stuff up into separate controls just for the sake of lowering indentation since I find it makes layouting a lot more annoying, especially when using the designer view.

Reply:
was more a preemptive "I don't want to argue" because the number always seems so arbitrary. I like 80 because it lets me fit stuff better on smaller screens and side by side on bigger ones. Obviously depends on fontsize as well and is just personal preference.

would still be nice to have autoformating so formatting nitpicks just aren't a thing

and yes, line length should be enforced for XAML or at least not 80 chars

Reply:
Pinged you on Discord about this a couple days ago, lets finish discussing formatting stuff there / in meeting today.